lib/output_helper.rb

```diff
@@ -120,6 +133,56 @@ def insert_visits(days, inserted_id, nb_visits)
       @scheduling_file << "#{line}\n"
     end
 
+    def compute_route(route_data)
```

Comment:
Can't we use the route_details function with the call to compute_batch to generate the route trace if the geometry is needed ? 
https://github.com/Mapotempo/optimizer-api/blob/c3e0b1732e85c95e21c0bf8982cc0b4f1704a1e5/optimizer_wrapper.rb#L676

Reply:
That is a good idea. However we would need to re-work this function a little bit because, for instance, scheduling would not want to send a route put rather a list of point_ids (by the way I think this function needs a rework anyway in order to use _activity.point.id_ instead of looking for corresponding service/shipment). Also, scheduling algo does not have access to _vrp_ at this point.

I have other modifications regarding output, maybe I can push them all later and taking these comments into account.